10-JavaScript/README.md

```diff
@@ -2,13 +2,12 @@
 
 ## Introduction
 1. [Welcome to JS](welcome-to-js.md)
-  - [Worksheet: Ruby to JS](ruby-to-js-worksheet.md)
-  
-1. [Fun with Functions](fun-with-functions.md)
-  - [Worksheet: Functions](functions-worksheet.md)
-  
-1. [Objects](objects.md)
-  - [Worksheet: Objects](objects-worksheet.md)
+  - [Worksheet: Ruby to JS](exercises/ruby-to-js-worksheet.md)
+2. [Fun with Functions](fun-with-functions.md)
+  - [Worksheet: Functions](exercises/functions-worksheet.md)
+3. [Classes](objects.md)
```

Comment:
is this supposed to classes.md?

Are we then not doing the objects.md discussion or do we need to add another link here?

Reply:
Fixed the link, I'm thinking only covering traditional JS objects with prototype briefly focusing on classes while leaving the objects lecture for those interested.